examples/pxScene2d/src/pxScene2d.h

```diff
@@ -1260,15 +1264,13 @@ class pxScene2d: public rtObject, public pxIView
 
   rtMethodNoArgAndNoReturn("dispose",dispose);
 
+  static void garbageCollect();
```

Comment:
Please remove this API under pxScene2d.  We can access this by calling script.garbageCollect() directly.